README.template.md

```diff
@@ -59,39 +59,42 @@ There are a number of `docker-compose.yml` files provided by this repository:
 
 ## Override
 
-This repository ignores `docker-compose.override.yml` which will be included in
-any `docker compose` commands you invoke by default.
+This git repository does not track `docker-compose.override.yml` which will 
```

Comment:
why? Are we supposed to put secret stuff there?

Reply:
This is intended to be the location where you drop your institution's specific overrides.

Reply:
Yes, but I just created a Git Repo to hold all my institution's customizations (so I thought). What was the purpose of doing that, if they're not to be included?